examples/01_prepare_data/mind_utils.ipynb

```diff
@@ -0,0 +1,528 @@
+{
```

Comment:
<p>Would you mind to move this function to the libraries? maybe in the mind utils <code>from reco_utils.dataset.mind</code>.  Would you please explain what the regex does in the docstring?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1247/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>